portfolio/src/main/webapp/smooth-scroll.js

```diff
@@ -0,0 +1,24 @@
+$(document).ready(function(){
```

Comment:
honestly, that's one of the stuff in my resources, looks like I got it from here: 

https://www.w3schools.com/howto/howto_css_smooth_scroll.asp

I pasted it in, it magically worked, so I just dropped them in the resources section and didn't ask any questions. This chunk is toward the bottom of that page

Reply:
Huh interesting, I don't see where `smooth-scroll.js` is being soured in the html file, could you point me to where it is? :o Sorry I might be looking in the wrong place

Reply:
Nope, you're in the right place. Long story short is that I confused myself :| but now have the implementation of that script working correctly 

Long story is:

In styles.css:
`html {
    scroll-behavior: smooth;  
  }`

Handles smooth scrolling, but this attribute doesn't work on safari, so that's why I decided to have that extra script to do smooth scrolling. When checking to see if that script worked, I didn't remove the scroll-behavior form styles.css, so the "working" behavior I was seeing was not from that script, but from the attribute in styles.css (as you saw, I didn't even have a reference to the script in the index.html 🤦. 

Fixed the implementation and it's working now.